src/analyzer.ts

```diff
@@ -198,15 +199,29 @@ export class BuildAnalyzer {
     //   1. The resolver stream, to resolve each file loaded via the analyzer
     //   2. The analyzer stream, to analyze app fragments for dependencies
     this._sourcesProcessingStream =
-        this._sourcesStream.pipe(new ResolveTransform(this))
+        this._sourcesStream
+            .on('error',
+                (err: Error) =>
+                    this._sourcesProcessingStream.emit('error', err))
+            .pipe(new ResolveTransform(this))
+            .on('error',
```

Comment:
Not sure I completely understand your question, but if you're asking "doesn't this end up re-emitting a `_sourcesStream` error already caught on L203?" the answer is that `pipe()` doesn't propagate errors, so the `ResolveTransform` stream would never see or re-emit an error that occurred in `this_sources` stream. (See https://github.com/nodejs/node/blob/master/lib/stream.js#L26)